### PR TITLE
Pimpin' ain't easy

### PR DESCRIPTION
**Pull request by DeemsW**
should be written as
pimp uneasily

### CODE REVIEW

**Comment by DeemsW** [comment]
This pull request adds a pimp-ass bio as requested in #3.

File: curious-monkey-bio [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Deems Bio
+
+**Name:** Deems Funkatronic Wilson
+**Occupation:** pimp-ass pimp
+**Location:** Yo Mama
```
Comment (DeemsW) [inline curious-monkey-bio:5]: Fuck yo mama

Comment (DeemsW) [inline curious-monkey-bio:5]: srsly

Comment (andrewwcaldwell) [inline curious-monkey-bio:5]: ambiguous location. please turn on location services for "Yo Mama"

**Pull request merged**